src/snippet/mod.rs

```diff
@@ -50,35 +53,51 @@ impl FragmentCandidate {
 }
 
 /// `Snippet`
-/// Contains a fragment of a document, and some highlighed parts inside it.
+/// Contains a fragment of a document, and some highlighted parts inside it.
 #[derive(Debug)]
 pub struct Snippet {
     fragment: String,
     highlighted: Vec<Range<usize>>,
+    highlighting_prefix: String,
+    highlighting_postfix: String,
 }
 
-const HIGHLIGHTEN_PREFIX: &str = "<b>";
-const HIGHLIGHTEN_POSTFIX: &str = "</b>";
-
 impl Snippet {
-    /// Create a new, empty, `Snippet`
+    /// Create a new `Snippet`.
+    fn new(
+        fragment: &str,
+        highlighted: Vec<Range<usize>>,
+        highlighting_prefix: &str,
```

Comment:
See other comment:

Remove this from the new constructor and move that to an accessor to remove noise from select_best_fragment_combination

Reply:
Removed and moved to Snippet